src/inference/tests/functional/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ ov_add_test_target(
         LINK_LIBRARIES
             gmock
             func_test_utils
+            sharedTestClasses
```

Comment:
@olpipi  Could you please check that `sharedTestsClasss` are still need. I have created this lib for serilization tests in `ov_inference_func_tests` to call `serialize` method. But Not sure that they still exist... In case these tetsts were removed, we can merge 2 libs: shared test classes and plugin_shared_tests